telemetry/definitions/commonDefinitions.json

```diff
@@ -2240,6 +2310,175 @@
                 }
             ]
         },
+        {
+            "name": "amazonq_unitTestGeneration",
+            "description": "Client side metrics of AmazonQ Unit Test Generation",
+            "metadata": [
+                {
+                    "type": "acceptedCharactersCount",
+                    "required": false
+                },
+                {
+                    "type": "acceptedCount",
+                    "required": false
+                },
+                {
+                    "type": "acceptedLinesCount",
+                    "required": false
+                },
+                {
+                    "type": "credentialStartUrl",
+                    "required": false
+                },
+                {
+                    "type": "cwsprChatProgrammingLanguage"
+                },
+                {
+                    "type": "executedCount",
+                    "required": false
+                },
+                {
+                    "type": "failedCount",
+                    "required": false
+                },
+                {
+                    "type": "generatedCharactersCount",
+                    "required": false
+                },
+                {
+                    "type": "generatedCount",
+                    "required": false
+                },
+                {
+                    "type": "generatedLinesCount",
+                    "required": false
+                },
+                {
+                    "type": "isSupportedLanguage"
+                },
+                {
+                    "type": "jobGroup",
+                    "required": false
+                },
+                {
+                    "type": "successCount",
+                    "required": false
+                },
+                {
+                    "type": "userEnteredPromptMessage"
```

Comment:
you cannot save this as telemetry.  This is customer data.

Reply:
As declared this is boolean, so there is no harm for customer data.